bower.json

```diff
@@ -1,15 +1,14 @@
 {
   "name": "cfgov-sheer-templates",
   "description": "Sheer templates for consumerfinance.gov",
-  "version": "0.3.1",
+  "version": "1.0.0",
```

Comment:
I get all that, but I think it's worth the potential confusion to have it in the code.

You're saying if `version` is in there, it checks for a mismatch and actually throws an error? Why not just throw a warning?